community/jansson/build

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+./configure --prefix=/usr --disable-static
+make PREFIX=/usr
+make DESTDIR="$1" PREFIX=/usr install
```

Comment:
These should be on newlines with a linebreak before the `make ...` lines. You also don't need to specify PREFIX for the `make` lines here.

```
./configure \
    --prefix=/usr \
    --disable-static

make
make DESTDIR="$1" install
```

Reply:
That's a good idea, the prefix wasn't really needed in the make commands considering it was defined while executing the configure script. I'll make those changes to the package, thank you!